### PR TITLE
Adds a brand-new HUD: The Trasen-Knox HUD!

### PR DESCRIPTION
**Pull request by Imaginos16**
## About The Pull Request
Hello everyone! Today I bring to you all a fancy new HUD for you lot to enjoy: The Trasen-Knox HUD! Lovingly based off of the Common Core lore maintained by @EOBGames!

Well, what is it?

image

Its a HUD with a special theming based on the old 1970's CRT screens that were commonplace at the time, including the iconic neon green and the beige plastic casing!

This PR also includes a new behavior so that its possible for each active hand icon to be unique, instead of it being a sole highlighted icon as it was before.
image

And for those wondering, it does indeed account for multiple limbs!
image

Finally, as seen in the first screenshot, this PR adds the behavior to hide the HUD icon when you insert a piece of equipment into the slot. Thank you for adding the behavior, @Fikou!

## Why It's Good For The Game
This is all as a step for upgrading and changing how the HUDs we interact with on a daily basis can be upgraded in the future to be more appealing visually! That, and it is simply gorgeous :)

## Changelog

:cl: PositiveEntropy, Fikou
add: Adds a new HUD: The Trasen-Knox HUD! With a theming based on classic 1970s CRT screen computers!
code: Added behavior for unique icons for active hands.
code: Added behavior that makes it so when you equip an item, the icon behind it defaults to the blank template type.
/:cl:

### CODE REVIEW

**Comment by FloFluoro** [comment]
Do you have a picture of the HUD running on Paracode? I'm assuming your provided picture is from /tg/ based on the lack of a second ear slot/PDA slot.

**Comment by YakumoChen** [comment]
Now all that's left is to give the health doll some greaser hair, add a 13 on the jumpsuits...

File: code/modules/mob/living/carbon/human/human_update_icons.dm [hunk 1]
```diff
@@ -452,8 +452,8 @@ There are several things that need to be remembered:
 		var/atom/movable/screen/inventory/inv
 
 		inv = hud_used.inv_slots[TOBITSHIFT(ITEM_SLOT_LPOCKET) + 1]
+		inv.update_icon()
 		inv = hud_used.inv_slots[TOBITSHIFT(ITEM_SLOT_RPOCKET) + 1]
-
 		inv.update_icon()
```
Comment (LemonInTheDark) [inline code/modules/mob/living/carbon/human/human_update_icons.dm:457]: what?

File: code/modules/mob/living/carbon/human/human_update_icons.dm [hunk 2]
```diff
@@ -452,8 +452,8 @@ There are several things that need to be remembered:
 		var/atom/movable/screen/inventory/inv
 
 		inv = hud_used.inv_slots[TOBITSHIFT(ITEM_SLOT_LPOCKET) + 1]
+		inv.update_icon()
```
Comment (LemonInTheDark) [inline code/modules/mob/living/carbon/human/human_update_icons.dm:455]: Doesn't need to exist, is covered by the second call
```suggestion
```

**Review by Twaticus — APPROVED** [review]

File: code/modules/mob/living/carbon/human/human_update_icons.dm [hunk 3]
```diff
@@ -453,7 +453,6 @@ There are several things that need to be remembered:
 
 		inv = hud_used.inv_slots[TOBITSHIFT(ITEM_SLOT_LPOCKET) + 1]
 		inv = hud_used.inv_slots[TOBITSHIFT(ITEM_SLOT_RPOCKET) + 1]
```
Comment (LemonInTheDark) [inline code/modules/mob/living/carbon/human/human_update_icons.dm:455]: damn you do need it I hate it here
```suggestion
		inv.update_icon()
		inv = hud_used.inv_slots[TOBITSHIFT(ITEM_SLOT_RPOCKET) + 1]
```

**Comment by Twaticus** [comment]
image
ill remember this

**Comment by Twaticus** [comment]
image
yeah thats right you better run

**Pull request merged**